virt/virtual_machines/advanced_vm_management/virt-working-with-resource-quotas-for-vms.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="virt-working-with-resource-quotas-for-vms"]
+= Working with resource quotas for virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-working-with-resource-quotas-for-vms
+
+toc::[]
+
+Create and manage resource quotas for virtual machines.
+
+include::modules/virt-setting-resource-quota-limits-for-vms.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Please use the add'l resources formatting for assemblies:

```
[role="_additional-resources"]
[id="additional-resources_virt-working-with-resource-quotas-for-vms"]
== Additional resources
```

Reply:
Done. Thanks.

Reply:
Thanks for adding the `role` and `id` lines! One last thing, though: you still have `.Additional resources` in line 15. That should be changed to `== Additional resources`.

Reply:
Sorry about that. Should be good now :-)